third_party/github/repos.bzl

```diff
@@ -0,0 +1,14 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+def github_tools_repos():
+    http_archive(
```

Comment:
looks like I will have to regen the airgapped image once this is in a CI-passing state.